packages-next/keystone/src/lib/createKeystone.ts

```diff
@@ -25,7 +25,7 @@ export function createKeystone(config: KeystoneConfig, dotKeystonePath: string,
     adapter = new MongooseAdapter({ mongoUri: db.url, ...db.mongooseOptions });
   } else if (db.adapter === 'prisma_postgresql') {
     adapter = new PrismaAdapter({
-      getPrismaPath: () => path.join(dotKeystonePath, 'prisma'),
+      getPrismaPath: () => '.keystone/prisma',
```

Comment:
```suggestion
      getPrismaPath: () => path.join(dotKeystonePath, 'prisma'),

```

Reply:
not resolving this path is important so that adapter-prisma will do it because `@keystone-next/keystone` will be bundled in the next app(but `@keystonejs` packages won't be, re: why just `@keystonejs`? because it's an easy line to draw because it's all backend code) and webpack will essentially mock `process.cwd()` to return `"/"` so without this change, the prisma adapter would try to write to `/.keystone/prisma`(as in at the root of the file system).

Reply:
@mitchellhamilton do you mean we _should_ merge @timleslie's suggestion? or not?

Reply:
I mean that we very much _shouldn't_